sources/api/ecs-settings-applier/src/main.rs

```diff
@@ -67,6 +73,8 @@ fn run() -> Result<()> {
             .iter()
             .map(|s| s.to_string())
             .collect(),
+        task_iam_role_enabled: true,
+        task_iam_role_enabled_for_network_host: true,
```

Comment:
If we're expecting to lift these into settings (in the near future), then let's add a comment saying that. Otherwise, the static tuning should be commented with our rationale behind em.

Reply:
I can add a comment.  I have no plans to lift these into settings unless someone actually wants to disable the feature.

Reply:
@jahkeup Done